python/lsst/ctrl/mpexec/cmdLineParser.py

```diff
@@ -91,6 +91,7 @@ def __call__(self, value):
 _ACTION_LABEL_TASK = _PipelineActionType("relabel", "(?P<label>.+):(?P<value>.+)")
 _ACTION_CONFIG = _PipelineActionType("config", "(?P<label>[^.]+)[.](?P<value>.+=.+)")
 _ACTION_CONFIG_FILE = _PipelineActionType("configfile", "(?P<label>.+):(?P<value>.+)")
+_ACTION_NAME_TEMPLATES = _PipelineActionType("name_templates", "(?P<label>[^.]+)[=](?P<value>.+)")
```

Comment:
Should this use `:` instead of `=` for consistency with other options?

Reply:
I really have no opinion on this, I had thought having the equals sign make it more akin to setting a config option, as you are kind of assigning something to that task, but then again it is a different pattern because normally the label is just that a label not something that is assignable. So maybe it would be good to go back to : as the command line switch name already indicates that you are doing an action.